openmc/mgxs/mdgxs.py

```diff
@@ -72,6 +78,10 @@ class MDGXS(MGXS):
         Energy group structure for energy condensation
     delayed_groups : list of int
         Delayed groups to filter out the xs
+    num_polar : None or Integral
```

Comment:
Ah I should have continued on :-).  All our parameters for MGXS (e.g.) seem to be a little more descriptive than the attributes, as are these.